[2021-01-15 16:32:10,880] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: noti_test.op3 2021-01-15T06:51:48.687195+00:00 [queued]>
[2021-01-15 16:32:10,889] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: noti_test.op3 2021-01-15T06:51:48.687195+00:00 [queued]>
[2021-01-15 16:32:10,889] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-01-15 16:32:10,889] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2021-01-15 16:32:10,889] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-01-15 16:32:10,893] {taskinstance.py:901} INFO - Executing <Task(PythonOperator): op3> on 2021-01-15T06:51:48.687195+00:00
[2021-01-15 16:32:10,895] {standard_task_runner.py:54} INFO - Started process 8961 to run task
[2021-01-15 16:32:10,936] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'noti_test', 'op3', '2021-01-15T06:51:48.687195+00:00', '--job_id', '54', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/noti_test.py', '--cfg_path', '/var/folders/4f/xnn880bn51d9945mdydbgwdw0000gn/T/tmpj9xs48mm']
[2021-01-15 16:32:10,937] {standard_task_runner.py:78} INFO - Job 54: Subtask op3
[2021-01-15 16:32:10,960] {logging_mixin.py:112} INFO - Running <TaskInstance: noti_test.op3 2021-01-15T06:51:48.687195+00:00 [running]> on host 1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa
[2021-01-15 16:32:10,970] {logging_mixin.py:112} INFO - function3
[2021-01-15 16:32:10,971] {python_operator.py:114} INFO - Done. Returned value was: None
[2021-01-15 16:32:15,887] {local_task_job.py:102} INFO - Task exited with return code Negsignal.SIGABRT
